server/api/users/users.controller.ts

```diff
@@ -19,43 +19,23 @@ export const getCurrentUser: RequestHandler<void, ClientUser>  = (req, res) => {
 };
 
 export const getUserById: RequestHandler<{id: string}, UserProfile> = async (req, res) => {
-  const user = await User.findByPk(req.params.id, {
-    attributes: [
-      'id',
-      'discord_user_id',
-      'discord_name',
-      'bio',
-      'twitter_username',
-      'linkedin_url',
-      'github_username',
-      'website',
-    ],
-  });
+  const allowedFields = User.allowedFields;
+  const user = await User.findByPk(req.params.id)
```

Comment:
I'd probably inline `allowedFields` in the query; you can then delete lines 22 and 29 then return `user` in the response.
```js
const user = await User.findByPk(req.params.id, { attributes: User.allowedFields });
```

Reply:
Doesn't this do away with _.pick? Won't @timmyichen smite me for going against his request?

Reply:
Since you now have the field names extracted to `User.allowedFields`, there's no need to `_.pick()` when you can pass `User.allowedFields` to Sequalize and have it only select those fields from the get-go.

Thus eliminating the need for extra variables, looping over the results to cherry-pick items, and importing lodash at all.